pymongo/asynchronous/topology.py

```diff
@@ -497,7 +497,7 @@ async def _process_change(
                     (td_old, self._description, self._topology_id),
                 )
             )
-        if _SDAM_LOGGER.isEnabledFor(logging.DEBUG):
+        if _SDAM_LOGGER.isEnabledFor(logging.DEBUG) and sd_old != server_description:
```

Comment:
Should this be ` and not suppress_event`?

Reply:
`suppress_event` checks for event publishing flags, which we don't use for logging.

Reply:
Could that be changed?
```
suppress_event = sd_old == server_description
```